runtime/jcl/common/java_dyn_methodhandle.c

```diff
@@ -55,9 +56,6 @@ static void JNICALL vmInvalidate(JNIEnv *env, jclass mutableCallSiteClass, jlong
 static BOOLEAN accessCheckMethodSignature(J9VMThread *currentThread, J9Method *method, j9object_t methodType, J9UTF8 *lookupSig);
 static BOOLEAN accessCheckFieldSignature(J9VMThread *currentThread, J9Class* lookupClass, UDATA romField, j9object_t methodType, J9UTF8 *lookupSig);
 static char * expandNLSTemplate(J9VMThread *vmThread, const char *nlsTemplate, ...);
-#if defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT)
-static void clearNonZAAPEligibleBit(JNIEnv *env, jclass nativeClass, const JNINativeMethod *nativeMethods, jint nativeMethodCount);
```

Comment:
This declaration needs to be in a .h file.

Reply:
Ah, I see it now.

Reply:
Yes, it is.